meson_options.txt

```diff
@@ -46,3 +50,7 @@ option('man-pages',
        type: 'feature',
        value: 'auto',
        description: 'Build man pages (requires rst2man)')
+option('sandboxed-image-validation',
+       type: 'boolean',
+       value: true,
+       description: 'Enables bubblewrap sandboxing for image validation. Disabling this option could lead to security vulnerabilities')
```

Comment:
`Use Bubblewrap to sandbox image validation. Disabling this option may lead to security vulnerabilities.`

Reply:
changed